Doppler.PushContact/Controllers/PushContactController.cs

```diff
@@ -48,5 +49,18 @@ public async Task<IActionResult> Get([FromRoute] string domain)
 
             return Ok(pushContacts);
         }
+
+        [HttpDelete]
+        public async Task<IActionResult> Delete([FromBody] IEnumerable<string> deviceTokens)
+        {
+            var deleteResult = await _pushContactService.DeleteByDeviceTokenAsync(deviceTokens);
+
+            if (deleteResult.MatchedCount <= 0)
```

Comment:
que tal si devolvemos del servicio un resultado de itemas afectados directamente (un int) y hacemos algo como
```suggestion
            return deleteResult == 0 ? NotFound() : Ok(); 
```

Reply:
Buenas sugerencias! Me parece bien en este momento retornar solo el número de afectados en el servicio, lo que sí deberíamos sacrificar la respuesta `NotFound()` ya que no necesariamente un número de afectados igual a cero se corresponde con no haber encontrado recursos. A mi entender deberíamos simplemente retornar `OK` (acá podemos ver de incluir el número de afectados retornado por el servicio) e `internal server error` ante excepciones. Lo hablamos!

Reply:
Se decidió eliminar el modelo `DeletePushContactsResult` y que el servicio solo retorne el número de elementos afectados. Por su parte, el controlador retornará dicho número como parte re una respuesta de éxito.